docs/patrickjmarion.md

```diff
@@ -0,0 +1,4 @@
+# Github patrickjmarion Bio 
+- **Name:** dude
+- **Occupation:**dreamer
+- **Location:**moon
```

Comment:
balmy during the day, a bit chilly at night.  Climate is otherwise pretty stable and predictable; except for one or two meteor here and there